src/Admin/Core/Extension/PublishableExtension.php

```diff
@@ -27,25 +27,41 @@ class PublishableExtension extends AbstractAdminExtension
      * @var string
      */
     protected $formGroup;
+    protected $formTab;
 
     /**
-     * @param string $formGroup - group to use for form mapper
+     * @param string $formGroup The group to use for form mapper
+     * @param string $formTab   The tab to use for form mapper
      */
-    public function __construct($formGroup = 'form.group_publish_workflow')
+    public function __construct($formGroup = 'form.group_publish_workflow', $formTab = 'form.tab_publish')
     {
         $this->formGroup = $formGroup;
+        $this->formTab = $formTab;
     }
 
     /**
      * {@inheritdoc}
      */
     public function configureFormFields(FormMapper $formMapper)
     {
+        if ($formMapper->hasOpenTab()) {
+            $formMapper->end();
+        }
+
         $formMapper
-            ->with($this->formGroup, array('translation_domain' => 'CmfCoreBundle'))
-                ->add('publishable', CheckboxType::class, array(
-                    'required' => false,
-                ))
+            ->tab($this->formTab, 'form.tab_publish' === $this->formTab
```

Comment:
what if i configure the publish to be in the form.tab_default (or whatever the default tab is called)? will this overwrite the tab, or add to the tab? which options win, the first or the last? or are they merged?